src/wallet/wallet.cpp

```diff
@@ -2859,7 +2859,7 @@ TransactionError CWallet::SignPSBT(PartiallySignedTransaction& psbtx, bool& comp
 
                         CConfidentialNonce nonce;
                         nonce.vchCommitment.insert(nonce.vchCommitment.end(), o.m_ecdh_pubkey.begin(), o.m_ecdh_pubkey.end());
-                        if (!UnblindConfidentialPair(blinding_key, o.m_value_commitment, o.m_asset_commitment, nonce, o.script.get(), o.m_value_rangeproof, value, value_factor, asset, asset_factor)) {
+                        if (UnblindConfidentialPair(blinding_key, o.m_value_commitment, o.m_asset_commitment, nonce, o.script.get(), o.m_value_rangeproof, value, value_factor, asset, asset_factor)) {
```

Comment:
In 96148142e8a575a0aa72fd7579aab6cfb01c4fb4 "wallet: fix "cannot unblind IsMine output" check in SignPSBT"

Is the comment below still true? It seems to me that by changing this check, the assertions below should always be triggered.

Reply:
No, the original code was wrong (and it was possible to trigger the assertions)